src/doc/src/reference/config.md

```diff
@@ -1361,6 +1362,13 @@ Controls whether or not progress bar is shown in the terminal. Possible values:
 
 Sets the width for progress bar.
 
+#### `term.progress.taskbar`
+* Type: bool
+* Default: auto-detect
+* Environment: `CARGO_TERM_PROGRESS_TASKBAR`
```

Comment:
Just a mild note, this environment variable doesn't work by itself (because it requires the other `progress` settings).

Reply:
BTW, you might be able to fix this by adding `#[serde(default)]` to the `ProgressConfig::when` field.

Reply:
Great point! Definitely more ergonomic now.

![image](https://github.com/user-attachments/assets/fe32c1ae-5f03-4eb2-8b1b-6432130baabf)
![image](https://github.com/user-attachments/assets/62335af9-638a-4ebc-8be7-ddf940db33ce)

Reply:
This breaks test `bad_progress_config_missing_when` 

I don't understand whe `ProgressWhen` has a `Default` implementation then...

Reply:
I don't recall any particular decisions made regarding `bad_progress_config_missing_when`. I personally would not object to changing that to allow it to pass. I can't think of a reason it would be required to specify `when` with `width`.